Header header

string object_class
float32 p

uint16 x
uint16 y

uint16 width
uint16 height